<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Метод иерархий</string>
  </property>
  <widget class="QWidget" name="mCentralWidget">
   <layout class="QVBoxLayout" name="mMainLayout" stretch="20,0,0,15">
    <property name="leftMargin">
     <number>75</number>
    </property>
    <property name="rightMargin">
     <number>75</number>
    </property>
    <item>
     <widget class="QTreeView" name="mTreeView"/>
    </item>
    <item>
     <widget class="QPushButton" name="mOkButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="__formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="__label">
        <property name="text">
         <string>Альтернативы</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="mAltsSpin">
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="mAltwidget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2">
        <widget class="QScrollArea" name="mScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="mAltsNameWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>370</width>
            <height>127</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="mAltsNameLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="__toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mInsertRowAction"/>
   <addaction name="mRemoveRowAction"/>
   <addaction name="mInsertChildAction"/>
  </widget>
  <action name="mInsertRowAction">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/pics/minus.png</normaloff>:/res/pics/minus.png</iconset>
   </property>
   <property name="text">
    <string>insertRow</string>
   </property>
   <property name="toolTip">
    <string>Ctrl + R</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="mRemoveRowAction">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/pics/plus.png</normaloff>:/res/pics/plus.png</iconset>
   </property>
   <property name="text">
    <string>removeRow</string>
   </property>
   <property name="toolTip">
    <string>Ctrl + X</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="mInsertChildAction">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/pics/network.svg</normaloff>:/res/pics/network.svg</iconset>
   </property>
   <property name="text">
    <string>insertChild</string>
   </property>
   <property name="toolTip">
    <string>Ctrl + E</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="mResetAction">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/pics/arrows-reset.png</normaloff>:/res/pics/arrows-reset.png</iconset>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
   <property name="toolTip">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
